core/chains/evm/config/chain_scoped.go

```diff
@@ -47,14 +44,10 @@ func (c *ChainScoped) BlockEmissionIdleWarningThreshold() time.Duration {
 func (c *ChainScoped) Validate() (err error) {
```

Comment:
@jmank88 I think this check will now fail in libocr. Is there another place we can move it to?

Reply:
What do you mean? I don't think we can change libocr types. Why are they tangled up in this?

Reply:
Turns out `LocalConfig` is used in two places, [here](https://github.com/smartcontractkit/chainlink/blob/develop/core/services/ocr/validate.go#L124) and [here](https://github.com/smartcontractkit/chainlink/blob/develop/core/services/ocr/delegate.go#L165), and they both call `ocr.SanityCheckLocalConfig()` afterwards, so I'd say it's ok if we simply remove it from chain_scoped.